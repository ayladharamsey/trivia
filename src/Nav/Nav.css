h1 {
    font-family: 'Courier New', Courier, monospace;
    font-weight: 600;
}

.navContainer {
    display: flex;
    justify-content: space-between;
    width:100vw;
    padding: 30px;
}
