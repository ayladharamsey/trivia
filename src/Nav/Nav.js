import React from 'react';
import PropTypes from 'prop-types';
import './Nav.css';
import { Link } from 'react-router-dom';

const Nav = ({ signOut }) => {
    return (
        <section className='navContainer'>
            <h1>Star Wars Trivia</h1>
            <Link to="/">
                <button onClick={signOut}>Sign Out</button>
            </Link>
        </section>
    )

}

export default Nav;

Nav.propTypes = {
    signOut: PropTypes.func,
  }
