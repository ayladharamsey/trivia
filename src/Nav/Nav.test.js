import React from 'react';
import { shallow } from 'enzyme';
import Nav from './Nav';

describe('Nav', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Nav 
      name='Ayla'
      quote='Shes pretty stellar'
      rating='master'
    />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });


});