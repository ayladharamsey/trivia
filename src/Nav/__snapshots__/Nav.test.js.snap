// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshot 1`] = `
<section
  className="navContainer"
>
  <h1>
    Star Wars Trivia
  </h1>
  <Link
    to="/"
  >
    <button>
      Sign Out
    </button>
  </Link>
</section>
`;
